frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -0,0 +1,182 @@
+# Automated Annotation of Single Cell Data (EXPERIMENTAL)
```

Comment:
In terms of doc site organization, I'm wondering if this should go into [section 5](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/frontend/doc-site/05__Annotate%20and%20Analyze%20Your%20Data/5_4__Annotating%20Data.mdx), which covers CELLxGENE Annotate (aka desktop)?

Also, if we move if there, readers might have trouble differentiating with the [Annotating Data](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/frontend/doc-site/05__Annotate%20and%20Analyze%20Your%20Data/5_4__Annotating%20Data.mdx) page. The title here definitely states "Automated Annotation", which is good, but maybe we need to change the _other_ page to explicitly state "Manual Annotation". Not really sure what's best, but want to call this out.

Reply:
Yes definitely, this doc was mistakenly created in this section - that move can be after all the current feedback has been solicited (otherwise, I'm guessing it messes with the diff).

Thanks for the callout @atolopko-czi, I think your suggestion is good - tagging @ambrosejcarr to see if he has any thoughts/opinions on the subject.